app/src/androidTest/java/ch/epfl/sdp/drone3d/service/drone/DroneExecutorTest.kt

```diff
@@ -56,7 +57,7 @@ class DroneExecutorTest {
 
         val droneData = DroneDataImpl(droneService)
 
-        val executor = DroneExecutorImpl(droneService, locationService, droneData)
+        val executor: DroneExecutor = DroneExecutorImpl(droneService, locationService, droneData)
```

Comment:
General comment for the file, you could init the services in one place instead of in every test

Reply:
As I'm mocking it, so changing its state, I don't want to have mocked data from older tests making it pass. Because then the order in which the tests are run would have an impact on them passing.